tests/dualtor/test_ipinip.py

```diff
@@ -126,7 +127,6 @@ def stop_garp(ptfhost):
 
     ptf_t1_intf = random.choice(get_t1_ptf_ports(tor, tbinfo))
     logging.info("send encapsulated packet from ptf t1 interface %s", ptf_t1_intf)
-    time.sleep(10)
```

Comment:
why remove the sleep here?

Reply:
We had added this sleep here in last commit but as this issue is seen only in mocked env, now we are moving it under the check for mocked dtor.

Reply:
@kevinskwang  - could you please check the comment from Anant?